server/test/mapping/mapping.test.js

```diff
@@ -0,0 +1,14 @@
+const app = require('../../app');
+const supertest = require('supertest');
+const request = supertest(app);
+
+// Mapping Config
+describe('GET /api/specification/mapping', () => {
+  it('Should return 200 status code', async () => {
```

Comment:
should return 200 is implementation detail, tests are like business requirements.
should return all conversion format or whatever we support? this is your chance to clarify what mapping means